tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "target": "ES5",
+    "outDir": "./dist",
+    "moduleResolution": "Node",
+    "lib": ["ES2015", "DOM", "DOM.Iterable"],
+    "typeRoots": ["@types"]
+  },
+  "include": ["./src/**/*.ts"],
+  "exclude": ["node_modules", "dist"],
+  "noImplicitAny": true
+}
```

`typeRoots` 부분을 `compilerOptions` 밖으로 빼 주셔야 동작합니다!
```suggestion
{
  "compilerOptions": {
    "allowJs": true,
    "target": "ES5",
    "outDir": "./dist",
    "moduleResolution": "Node",
    "lib": ["ES2015", "DOM", "DOM.Iterable"]
  },
  "typeRoots": ["@types"],
  "include": ["./src/**/*.ts"],
  "exclude": ["node_modules", "dist"],
  "noImplicitAny": true
}
```